spot-controller/ios/spot-controller/Info.plist

```diff
@@ -38,15 +38,15 @@
 		</dict>
 	</dict>
 	<key>NSBluetoothPeripheralUsageDescription</key>
-	<string>Detect nearby meeting devices</string>
+	<string>Detect nearby Spot devices</string>
```

Comment:
is this string updated by branding if there is any? if not it needs  to stay as it was

Reply:
Yep, I wrote them back, thanks